tests/virt_autotest/prepare_alp_host.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: This file does the necessary setup and checks for alp host to do virtualization test.
+# Maintainer: alice <xlai@suse.com>, qe-virt@suse.de
+
+package prepare_alp_host;
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+
+    # Validate that kernel-default is installed and kvm module is loaded.
+    # Note: the alp host unattended installation will include kernel-default
+    #       package installation. See poo#118939 for more details.
+    assert_script_run(q@rpm -qa | grep kernel-default-[0-9]@);
+    save_screenshot;
+    assert_script_run(q@lsmod |grep kvm@);
+    save_screenshot;
```

Comment:
No more comments about the entire PR, just duplicate code statements can be removed, such as save_screenshot() after assert_script_run() at here. assert_script_run() takes screenshot automatically, so save_screenshot() is redundent.

Reply:
From testapi.pm, I do not see direct save_screen, maybe hiding deeper in distribution code. Anyway, if the screenshot is captured there, the save_screenshot can be saved.

Reply:
Updated.